app/cli/cmd/attestation_init.go

```diff
@@ -81,6 +81,10 @@ func newAttestationInitCmd() *cobra.Command {
 				return newGracefulError(err)
 			}
 
+			if res.DryRun {
+				logger.Info().Msg("The attestation is being crafted in dry-run mode. It will not get stored once rendered")
```

Comment:
this is just removing the dry-run output from stdout and move it to stderr as we are doing lately in the rest of the code.